app/src/main/kotlin/com/heartlessveteran/myriad/navigation/Destinations.kt

```
@@ -0,0 +1,244 @@
+            } catch (e: IllegalArgumentException) {
+                false
+            }
+        } ?: true // Allow null
+    }
+    
+    fun validateSettingsSection(section: String?): Boolean {
+        return section?.let {
+            try {
+                SettingsSection.valueOf(it.uppercase())
+                true
+            } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)